source/basic.tex

```diff
@@ -1404,11 +1395,7 @@
 
 \pnum
 For the members of a class \tcode{X}, a name used
-in a member function body,
```

Comment:
Whoops, we meant "function body" here, not "member function body". This is technically a normative change, but it's obvious that the new wording is correct.

Reply:
So, we're good here, right?

Reply:
Yes.